packages/components/src/FormField/FormFieldTypes.ts

```diff
@@ -121,7 +122,7 @@ export interface CommonFormFieldProps {
    * clearable. if the input value isn't editable (i.e. `InputTime`) you can
    * set it to `always`.
    */
-  readonly clearable?: "never" | "always";
+  readonly clearable?: Clearable;
```

Comment:
further down, we accept 3 values
https://github.com/GetJobber/atlantis/blob/master/packages/hooks/src/useShowClear/useShowClear.ts#L1

and we have code to handle that case and do something different

https://github.com/GetJobber/atlantis/blob/master/packages/hooks/src/useShowClear/useShowClear.ts#L28-L29

we just weren't leveraging that same type and instead had a more restrictive type here

Reply:
oh no. something is not working with `focused` on other input types other than `InputText` 😬 quite possibly something I broke with my recent PR

that concept is only used in conjunction with `clearable` `while-editing`

I can revert those type changes so it's back to only `always | never` OR I can keep it like this which works on `InputText` but not others. technically we'd be slightly better off as far as functionality since now 1 type can use `while-editing` rather than 0. it's just that it's more obvious that the prop isn't working on other input types